src/main/java/uk/gov/companieshouse/service/dissolution/chips/ChipsResponseService.java

```diff
@@ -12,26 +13,31 @@
 public class ChipsResponseService {
 
     private final DissolutionRepository repository;
-
     private final DissolutionVerdictMapper dissolutionVerdictMapper;
-
     private final DissolutionEmailService dissolutionEmailService;
+    private final Logger logger;
 
     public ChipsResponseService(
             DissolutionRepository repository,
             DissolutionVerdictMapper dissolutionVerdictMapper,
-            DissolutionEmailService dissolutionEmailService
+            DissolutionEmailService dissolutionEmailService,
+            Logger logger
     ) {
         this.repository = repository;
         this.dissolutionVerdictMapper = dissolutionVerdictMapper;
         this.dissolutionEmailService = dissolutionEmailService;
+        this.logger = logger;
     }
 
     public void saveAndNotifyDissolutionApplicationOutcome(ChipsResponseCreateRequest body) {
         DissolutionVerdict dissolutionVerdict = this.dissolutionVerdictMapper.mapToDissolutionVerdict(body);
 
         Dissolution dissolution = this.repository.findByDataApplicationReference(body.getSubmissionReference()).get();
+
         dissolution.setVerdict(dissolutionVerdict);
+
+        logger.info(String.format("Received CHIPS verdict for company %s - %s", dissolution.getCompany().getNumber(), dissolutionVerdict.getResult().toString()));
```

Comment:
Some additional logging for chips verdict, adding this while im making the hotfix for simplicity.